smaug/python/ops/common.py

```diff
@@ -47,12 +47,17 @@ def add_node(
   # that we don't need to create a DataOp for input_data.
```

Comment:
Can you update this comment - "we don't need to create a DataOp for input_data"? What does that mean?

Reply:
Updated. This meant "don't create a data op for data op" to avoid deadlock.